packages/docz-tools/src/gatsby-theme-docz/components/Navigation/styles.ts

```diff
@@ -0,0 +1,76 @@
+export const navigation = {
+  pt: "base",
```

Comment:
We talked about expanding these shorthands to their more verbose form. I think that would still make this more readable.

Reply:
Do you think we should look at that, or handle this in a backlog item to remove theme-ui